File: class_4/class_4/app/src/main/java/com/alejouribesanchez/class_4/Main6Activity.java
package com.alejouribesanchez.class_4;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.RelativeLayout;

public class Main6Activity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main6);

        RelativeLayout relativeLayout = (RelativeLayout) findViewById(R.id.rl_activity6);
        relativeLayout.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(Main6Activity.this, Main7Activity.class);
                startActivity(intent);
            }
        });
    }
}

File: class_4/class_4/app/src/main/java/com/alejouribesanchez/class_4/Main6Activity.kt
package com.alejouribesanchez.class_4

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.RelativeLayout

class Main6Activity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main6)

        val relativeLayout = findViewById(R.id.rl_activity6) as RelativeLayout
        relativeLayout.setOnClickListener {
            val intent = Intent(this@Main6Activity, Main7Activity::class.java)
            startActivity(intent)
        }
    }
}
